dwave/cloud/computation.py

```diff
@@ -25,21 +35,37 @@
 
 @functools.total_ordering
 class Future(object):
```

Comment:
I think it's important to note that users should *never* create `Future` objects manually. They would be "dead", since problem submittal is done in Client, initiated by Solver. Messy for now.